server/src/main/java/io/crate/analyze/AnalyzedColumnDefinition.java

```diff
@@ -637,4 +639,8 @@ void setStorageProperties(GenericProperties<T> storageProperties) {
     public boolean docValues() {
         return docValues;
     }
+
+    public List<String> sources() {
+        return List.copyOf(sources);
```

Comment:
Why do we need to copy the list?

Reply:
We don't, I think I messed up when planning future usage. We agreed to stream References on CREATE TABLE, so this will be used only to resolve actual references later on in the next PR-s.